starter-code/index.html

```diff
@@ -5,40 +5,76 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Spotify</title>
+    <link rel="stylesheet" href="css/main.css">
+    <link href="https://fonts.googleapis.com/css?family=Varela+Round&display=swap" rel="stylesheet">
   </head>
   <body>
-    Premium Discover Help Download
-
-    Music for everyone.
-    Spotify is now free on mobile, tablet and computer.
-    Listen to the right music, wherever you are.
-
-    What’s on Spotify?
-
-    Millions of Songs
-    There are millions of
-    songs on Spotify
-
-    HD Music
-    Listen to music as if you
-    were listening live
-
-    Stream Everywhere
-    Stream music on your
-    smartphone, tablet or
-    computer
-
-    It’s as yeezy as Kanye West.
-    Search
-    Know what you want to listen to?
-    Just search and hit play.
-    Browse
-    Check out the latest charts,
-    brand new releases and great
-    playlists for right now.
-    Discover
-    Enjoy new music every Monday
-    with your own personal playlist.
-    Or sit back and enjoy Radio.
+    <nav class="nav-bar">
+      <div><img src="images/spotify-logo.png" alt="spotify-logo"></div>
+      <ul>
+        <li>Premium</li>
+        <li>Discover</li>
+        <li>Help</li>
+        <li>Download</li>
+      </ul>
+    </nav>
+    <header class="header">
+    <div class="conteiner">
+      <h1>Music for everyone.</h1>
+      <h2>Spotify is now free on mobile, tablet and computer.</h2>
+      <h2>Listen to the right music, wherever you are.</h2>
+    </div>
+    </header>
+    <!--section 1-->
+    <section id="section1">
+   
+      <h3>What’s on Spotify?</h3>
+      <div class="icons">
+        <article class="service-box">
+          <img src="images/music-icon.png" alt="music-icon">
+          <h4>Millions of Songs</h4>
+          <p>There are millions of</p>
+          <p>songs on Spotify</p>
+        </article>
+        <article class="service-box">
+          <img src="images/high-quality-icon.png" alt="quality-icon">
+          <h4>HD Music</h4>
+          <p>Listen to music as if you</p>
+          <p>were listening live</p>
+        </article>
+        <article class="service-box">
+          <img src="images/devices-icon.png" alt="devices-icon">
+          <h4>Stream Everywhere</h4>
+          <p>Stream music on your</p>
+          <p>smartphone, tablet</p><p>or computer</p>
+        </article>
+      </div>
+    </section>
+    <!--section 2-->
+    <section class="green-box">
+      <div class="text-box">
+        <h3>It’s as yeezy as Kanye West.</h3>
+        <article>
+          <h4>Search</h4>
+          <p>Know what you want to listen to?</p>
+          <p>Just search and hit play.</p>
+        </article>
+        <article>
+          <h4>Browse</h4>
+          <p>Check out the latest charts,</p>
+          <p>brand new releases and great</p>
+          <p>playlists for right now.</p>
```

Comment:
Here you are creating multiple "p" to put them on below another, you should in theory only have 1 p and the text would wrap according to it's container.